_data-prepper/pipelines/configuration/processors/mutate-event.md

```diff
@@ -0,0 +1,302 @@
+
+### Configuration
+
+You can configure the `copy value` processor with the following options.
+
+Option | Required | Description 
+:--- | :--- | :---
+| `entries` | Yes | A list of entries to be copied in an event. |
+| `from_key` | Yes | The key of the entry to be copied.
+| `to_key` | Yes | The key of the new entry to be added.
+| `overwrite_if_to_key_exists` | No | When set to a value of `true`, if `to_key` already exists in the event, then the existing value will be overwritten. The default value is `false`. |
+
+### Usage
+
+To get started, create the following `pipeline.yaml` file:
+
+```yaml
+pipeline:
+  source:
+    file:
+      path: "/full/path/to/logs_json.log"
+      record_type: "event"
+      format: "json"
+  processor:
+    - copy_values:
+        entries:
+        - from_key: "message"
+          to_key: "newMessage"
+          overwrite_if_to_key_exists: true
+  sink:
+    - stdout:
+```
+
+Next, create a log file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` with this filepath. For more information, see [Configuring Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/#2-configuring-data-prepper).
+
+```json
+{"message": "value"}
+```
+
+When you run this processor, it parses the message into the following output:
+
+```json
+{"message": "value", "newMessage": "value"}
+```
+
+> If `newMessage` had already exists, its current value is overwritten with `value`.
+
+
+## DeleteEntry
+
+The `DeleteEntry` processor deletes entries in an event, such as key-value pairs. You can define the keys you want to delete in the `with-keys` field following `delete_entries` in the YAML configuration file. Those keys along with their values are deleted. See the example below in the Usage section, where `with_keys: ["message"] exists in the YAML file. This means that the entry {"message": "value"} is deleted. 
+
+### Configuration
+
+You can configure the `DeleteEntry` processor with the following options.
+
+Option | Required | Description
+:--- | :--- | :---
+| `with_keys` | Yes | An array of keys of the entries to be deleted. | 
+
+### Usage
+
+To get started, create the following `pipeline.yaml` file:
+
+```yaml
+pipeline:
+  source:
+    file:
+      path: "/full/path/to/logs_json.log"
+      record_type: "event"
+      format: "json"
+  processor:
+    - delete_entries:
+        with_keys: ["message"]
+  sink:
+    - stdout:
+```
+
+Next, create a log file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` with this filepath. For more information, see [Configuring Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/#2-configuring-data-prepper).
+
+```json
+{"message": "value", "message2": "value2"}
+```
+
+When you run the `DeleteEntry` processor, it parses the message into the following output:
+
+```json
+{"message2": "value2"}
+```
+
+> If `message` does not exist in the event, then no action occurs.
+
+
+## RenameKey
```

Comment:
Which is correct: "RenameKey" or "rename key"?

Reply:
Should be "Rename Key".